src/ConfigEditor/ConfigEditor.tsx

```diff
@@ -114,13 +145,15 @@ export function ConfigEditor(props: Props) {
         saveOptions={saveOptions}
         hidden={!useManagedSecret}
       />
-      <InlineInput
+      <ConfigSelect
         {...props}
         value={props.options.jsonData.clusterIdentifier ?? ''}
-        onChange={onChange('clusterIdentifier')}
+        onChange={onChangeClusterID}
+        fetch={fetchClusters}
         label={selectors.components.ConfigEditor.ClusterID.input}
         data-testid={selectors.components.ConfigEditor.ClusterID.testID}
-        disabled={useManagedSecret}
+        saveOptions={saveOptions}
+        hidden={useManagedSecret}
```

Comment:
we should still show this field as disabled when `useManagedSecret`

Reply:
weird, when I have `disabled={useManagedSecret}` it does not disable the select. 

When I inspect, I see that although `ConfigSelect` has `disabled=true` it's child `ResourceSelector` has `disabled=false`. 

It seems to be because we override disable there: https://github.com/grafana/grafana-aws-sdk-react/blob/6a051089daf5c577297f4bf695909b4da0194f9f/src/sql/ConfigEditor/ConfigSelect.tsx#L47 

I will fix that as part of the SDK PR 🙂